app/gui/src/project-view/components/MarkdownEditor/MarkdownEditorImpl.vue

```diff
@@ -55,6 +58,9 @@ defineExpose({
     const pos = editorView.posAtCoords(coords, false)
     putTextAt(text, pos, pos)
   },
+  toggleHeader,
+  toggleQuote,
+  toggleList,
```

Comment:
`DocumentationEditor` is the component that integrates the markdown editor with the graph editor. Pure markdown-editing functionality should be in the `MarkdownEditorImpl`, so that the dashboard can use it too (once they enable editing as planned).

We should render the toolbar in this component. Instead of exposing the formatting functions, we can use slots to inject the functionality that isn't shared between graph editor and dashboard (fullscreen, insert-image).

Reply:
Done